transposer/plugins/org.polarsys.kitalpha.transposer.rules.handler.mapping.editor/generated/org/polarsys/kitalpha/transposer/rules/handler/rules/common/presentation/CommonEditor.java

```diff
@@ -45,6 +45,7 @@
 import org.eclipse.core.runtime.NullProgressMonitor;
 import org.eclipse.core.runtime.NullProgressMonitor;
 import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.core.runtime.NullProgressMonitor;
```

Comment:
Can you please clean these multiple imports ?

Reply:
ok